app/routes/schemas/application-identifier.js

```diff
@@ -0,0 +1,29 @@
+const Joi = require('joi')
+
+const LENGTH = 10
+
+module.exports = {
+  applicationIdentifier: Joi.string().regex(/^\w{10}$/).required()
```

Comment:
Use of regex as placeholder for any business logic in this that we discover later